src/app/pages/FastBtcPage/components/Deposit/SignatureValidation/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { isValidElement, useContext } from 'react';
```

Comment:
```suggestion
import React, { useContext } from 'react';
```

Remove unused import